src/configuration.cpp

```diff
@@ -43,4 +44,10 @@ IceServer::IceServer(const string &hostname_, const string &service_, string use
     : hostname(hostname_), service(service_), type(Type::Turn), username(username_),
       password(password_), relayType(relayType_) {}
 
+Configuration::Configuration(const LogLevel logLevel_) {
+	static plog::ColorConsoleAppender<plog::TxtFormatter> consoleAppender;
+	plog::init(static_cast<plog::Severity>(logLevel_), &consoleAppender);
```

Comment:
This sets the logger for the entire library, correct? What if you create different peer connections with different configurations?
My understanding is that log initialization should be a static function, maybe a function directly under the `rtc` namespace?
Also it could be cool to have an optional log appender to be able to pipe the log the the main program, like shown here: https://github.com/SergiusTheBest/plog#chained-loggers

Reply:
Setting logger for each config (or peer) is a complex situation.
I moved init task to rtc namespace.